Slider/Assets/Scenes/Village.unity

```diff
@@ -471497,15 +472372,16 @@ PrefabInstance:
       objectReference: {fileID: 0}
     - target: {fileID: 3652435011426292518, guid: eb8cd11432469439eb214403b87b5b92, type: 3}
       propertyPath: conds.Array.data[0].dialogueChain.Array.data[0].dialogue
-      value: Help! My buoys have gotten tangled up. Can you help untangle them?
+      value: Help! My buoys have gotten tangled up. Mind untanglin' them for me?
       objectReference: {fileID: 0}
     - target: {fileID: 3652435011426292518, guid: eb8cd11432469439eb214403b87b5b92, type: 3}
       propertyPath: conds.Array.data[1].dialogueChain.Array.data[0].dialogue
-      value: Thanks! I'll be at the beach some time soon.
+      value: Thanks a bundle! I don't have much, but you can have this strange junk
+        i fished up.
       objectReference: {fileID: 0}
     - target: {fileID: 3652435011426292518, guid: eb8cd11432469439eb214403b87b5b92, type: 3}
       propertyPath: conds.Array.data[1].dialogueChain.Array.data[1].dialogue
-      value: Rumor is there are new fish in the water.
+      value: I'm heading to the beach soon. Rumor is there are new fish in the water.
```

Comment:
Maybe he can be waiting for Pierre? Explains why he's not at the beach yet. Also they are partners (gay, not buisness)

Reply:
prob fine as is